presto-main/src/main/java/com/facebook/presto/operator/aggregation/builder/SpillableHashAggregationBuilder.java

```diff
@@ -278,9 +278,10 @@ private void rebuildHashAggregationBuilder()
         if (hashAggregationBuilder != null) {
             hashCollisions += hashAggregationBuilder.getHashCollisions();
             expectedHashCollisions += hashAggregationBuilder.getExpectedHashCollisions();
+            hashAggregationBuilder.close();
```

Comment:
make it `null` after you close it

Reply:
IMO this a good convention when object got invalidated (closed), because:
 - it makes it eligible for GC
 - it make close idempotent (safe for double close)

Nevermind, up to you.

Reply:
This is not close, but rather `rebuildHashAggregationBuilder()` method and `hashAggregationBuilder` reference is overwritten with new object immediately afterward, so explicit nulling would make sense only for exceptional cases which I choose not to optimize for.

Reply:
Sounds good